internal/mmf/harness.go

```diff
@@ -1,14 +0,0 @@
-package mmf
-
-import (
-	"log"
-
-	"github.com/GoogleCloudPlatform/open-match/config"
-	api "github.com/GoogleCloudPlatform/open-match/internal/pb"
-)
-
```

Comment:
This file got deleted

Reply:
Not sure what you mean. My change is deleting this file since it is not used anywhere (and current master has this file - so its not already been deleted).